js/spec/index-spec.js

```diff
@@ -1,7 +1,8 @@
 const nsfw = require('../src/');
 const path = require('path');
 const promisify = require('promisify-node');
```

Comment:
It looks like we're still using `promisify-node` here, but you mentioned removing it in the PR body and I see it removed from the dependencies. Was this an oversight?